test/javascripts/models/user-drafts-test.js.es6

```diff
@@ -0,0 +1,31 @@
+import UserDraft from "discourse/models/user-draft";
+import { NEW_TOPIC_KEY } from "discourse/models/composer";
+
+QUnit.module("model:user-drafts");
+
+QUnit.test("stream", assert => {
+  var user = Discourse.User.create({ id: 1, username: "eviltrout" });
```

Comment:
We generally prefer to use `let` over `var` as the variable will then be limited to the scope that it was declared in.

Reply:
in this case it should even be a `const` doesn’t seem mutated later.